Dockerfile

```diff
@@ -1,4 +1,14 @@
-FROM node:12-alpine AS build-stage
+FROM node:16-alpine AS build-stage
```

Comment:
I see you upgraded node version here as well which is good to follow the other repositories but not sure if we should add the engines in the package.json and add the .npmrc file to prevent using lower node versions. Or maybe that could happen when we do the package upgrades for the factory (I think we have a ticket for that in Jira).

Reply:
I think we should add it, but as a separate PR